xAPI.md

```diff
@@ -675,6 +675,18 @@ LRS SHOULD* reject Statements containing such additional properties.
 
 <a name="statement-immutablity-and-exceptions" />
 
+Statements are evidence of a tracked learning experience.  Because this information is considered to be evidence, 
+the information represented in the Statement has already happened.  Thus, Statements will use the past tense of 
+Verbs when they display information and IRIs are encouraged to follow this same pattern of tense whenever possible.
```

Comment:
Per the 7/22 call - we need to determine if it is a best practice if when verb "tokens" are used in an IRI, if they should be past tense for consistency....or is this getting too much into dictating what CoPs and profiles should do?